controllers/controllerArticles.js

```diff
@@ -29,3 +29,12 @@ exports.getArticles = (req, res, next) => {
     })
 }
 
+exports.getComments = (req, res, next) => {
+    const {article_id} = req.params
+    fetchComments(article_id).then((comment) => {
+        res.status(200).send(comment)

```

Comment:
send this back on a key of comments, {comments} then change your test to accommodate, it would be on body.comments